committee-steering/governance/README.md

```diff
@@ -1,23 +1,26 @@
-# SIG Governance Template
+# SIG Charter Guide
 
```

Comment:
A link to the requirement that SIGs should have a charter would be extremely helpful here to contextualize what the purpose of the rest of the doc is.

Reply:
Added a short section about this at the top.